rhel7/profiles/ospp.profile

```diff
@@ -1,438 +1,183 @@
 documentation_complete: true
 
-title: 'United States Government Configuration Baseline'
+title: 'Protection Profile for General Purpose Operating Systems'
 
 description: |-
```

Comment:
Please keep the description and just bump the OSPP version, or does the additional text fail NIAP validation?